_docs_integrate/requests-via-relationshiptemplates.md

```diff
@@ -56,77 +65,93 @@ For an overview of the available [RequestItems]({% link _docs_integrate/data-mod
-But note, that the `onNewRelationship` property is required and must, therefore, always be set.
+Next, the Sender wants to create the RelationshipTemplate, which contains the Request it wants to send to the Recipient. To specify a Request within a RelationshipTemplate, a data object of type [RelationshipTemplateContent]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplatecontent) must be used within the `content` property of the [RelationshipTemplate]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplate).
+A Request can be specified in the `onNewRelationship` property or the `onExistingRelationship` property of the RelationshipTemplateContent.
+If no Relationship has yet been established between the Sender and the Recipient, the Recipient receives the Request specified in the `onNewRelationship` property when loading the RelationshipTemplate.
+However, if an active Relationship already exists, the Recipient receives the Request specified in the `onExistingRelationship` property when loading the RelationshipTemplate, if such is specified. The specification of a Request in the `onNewRelationship` property is mandatory in contrast to the specification of a Request in the `onExistingRelationship` property.
+
+Note that the same Request can be specified in the `onExistingRelationship` property as in the `onNewRelationship` property, but a different Request can also be used.
+These customization options are useful as the Recipient that will load the RelationshipTemplate may not be known in advance.
+It is therefore possible that a Relationship between the Sender and the Recipient already exists or not.
+It is also possible to configure the RelationshipTemplate so that it can be loaded by multiple Identities.
+The creator of the RelationshipTemplate may have a Relationship to some of these Identities and no Relationship to others of these Identities.
+{: .notice--info}
+
+To create a RelationshipTemplate, the instructions of the [Create own RelationshipTemplate]({% link _docs_use-cases/use-case-transport-create-own-relationshiptemplate.md %}) use case documentation must be followed.
+A RelationshipTemplateContent needs to be specified in the `content` of the payload because the Sender wants to send the Recipient a Request via the RelationshipTemplate.
+In the payload example below, the [Request whose validity we have already checked](#check-the-requests-validity) is contained both within the `onNewRelationship` property and within the `onExistingRelationship` property of the [RelationshipTemplateContent]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplatecontent).
 
 ```jsonc
 {
   "maxNumberOfAllocations": 1,
-  "expiresAt": "2025-01-01T00:00:00.000Z",
+  "expiresAt": "2026-01-01T00:00:00.000Z",
   "content": {
+    // RelationshipTemplateContent
     "@type": "RelationshipTemplateContent",
-    "title": "Requests via RelationshipTemplates example",
```

Comment:
why is title removed?

Reply:
To keep the example minimal, I have removed the optional property `title`, as we often omit optional properties in the example payloads in the documentation.

Reply:
I think this is quite helpful as you see all kinds of properties of the request and not only the items.

Reply:
I have no problem with adding the property again 👍